modules/internal/ChapelArray.chpl

```diff
@@ -3263,7 +3263,9 @@ module ChapelArray {
       if !a.stridable && b.stridable then
         compilerError("cannot assign from a stridable domain to an unstridable domain without an explicit cast");
 
-    a._instance.dsiAssignDomain(b, lhsPrivate=false);
+    // distributions/robust/arithmetic/resizing/resizeFromOtherLocale.chpl
+    // fails if this uses b._instance instead of b._value
+    a._value.dsiAssignDomain(b, lhsPrivate=false);
```

Comment:
David, can you verify that lhs._arrs is indeed always empty if lhs is privatized? It may be that the ``resizeFromOtherLocale`` test isn't as robust as it ought to be.